### DISCUSSION TITLE
Error "skipped XXX/1708423791909.jpg due to invalid extension (unknown media type)" when scanning my photo library

### DISCUSSION THREAD

**Discussion by coutadeurf** [photoprism/photoprism#4456] [Q&A] [answered]
#### 1. What is not working as documented?

When indexing my library of more than 11000 photos, I got 30ish errors of jpeg files not being recognized, when this files are perfectly visible in Windows and get the right media type with all online tools I found (media/jpeg)

#### 2. How can we reproduce it?

1. Add the following file to your originals directory: ![1708423791909](https://github.com/user-attachments/assets/b771b175-dfae-4884-b2d6-fb9111567624)
2. Index your files
3. you will get the "unknown media type" error

#### 3. What behavior do you expect?

Index the jpg image

#### 4. What could be the cause of your problem?

After removing the metadata in Windows, the file is correctly indexed. If I compare the metadata before and after, they seems identical so I expect the metada to contains garbage somewhere

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

![1708423791909](https://github.com/user-attachments/assets/b771b175-dfae-4884-b2d6-fb9111567624)

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64 docker container in Proxmox LXC, Build [240528-977d6c0de]

(b) Database Type & Version: MariaDB, as stated in the docker compose file...

(c) Operating System Types & Versions: Linux (alpine)

(d) Browser Types & Versions: Firefox

(e) Ad Blockers, Browser Plugins, and/or Firewall Software? N/A

#### 7. On what kind of device is PhotoPrism installed?

(a) Device / Processor Type: Intel N100

(b) Physical Memory & Swap Space in GB: 4Gb & 2Gb

(c) Storage Type: SSD

(d) Anything else that might be helpful to know?

**Comment by lastzero** [comment, marked as answer]
It seems to work perfectly on our public demo (except that it's under review by default if the quality filter is enabled):
![IMG_8011](https://github.com/user-attachments/assets/7a80675b-c3e6-4155-aa1b-5e1e568400ba)

If you still believe it's a bug that we must fix in our software, please use our latest stable release for testing, enable debug logs, and include all relavant server logs before filling an official bug report. Since nobody else has issues with JPEG files, I believe it's more likely a local problem e.g. with the filesystem. Thank you very much!

**Reply by lastzero** [reply]
Also make sure the file extension doesn't contain any whitespace or other uncommon / invisible / invalid characters, and that the filename is properly UTF8 encoded.

**Reply by coutadeurf** [reply]
My bad; my docker image was stuck in mai (wheras the public demo has been updated less than one week ago). I will update my image and test again. Thanks!

**Comment by ledlamp** [comment]
I encountered this problem, it turns out the photos were mode 600 for some reason and not readable by photoprism.

I make sure all permissions are correct with `chmod -R a=r,u+w,a+X <photos folder>` and now it works.

Photoprism should give a more useful error that says it doesn't have permission to read the file instead of "invalid extension (unknown media type)" which is confusing and misleading.
